diagram_analysis/scope_plan.py

```
@@ -0,0 +1,172 @@
+        component.component_id
+        for component in scope.components
+        if component.component_id
+        and any(
+            method.qualified_name in changed_members for group in component.file_methods for method in group.methods
+        )
+    }
+
+    operations: list[ScopeOperation] = []
+    kept: set[str] = set()
+    untouched = 0
+    for group, owner in zip(grouping.groups, grouping.owners):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)